src/reanimated2/animations.js

```diff
@@ -48,6 +48,36 @@ export function transformAnimation(animation) {
   animation.startValue = transform(animation.startValue, animation);
 }
 
+export function removeAnimationUnit(animation, unit) {
+  'worklet';
+  if (!animation) {
+    return;
+  }
+  animation.toValue = parseFloat(animation.toValue);
+  animation.current = parseFloat(animation.current);
+  animation.startValue = parseFloat(animation.startValue);
+}
+
+export function addAnimationUnit(animation, unit) {
+  'worklet';
+  if (!animation) {
+    return;
+  }
+  animation.toValue = animation.toValue + unit;
+  animation.current = animation.current + unit;
+  animation.startValue = animation.startValue + unit;
+}
+
+export function getUnit(value) {
+  'worklet';
+  for (const unit of ['%', 'pt']) {
```

Comment:
maybe it would be good to define this array somewhere as, for instance, `ALLOWED_UNITS`?

Reply:
`pt` works only for web. RN supports only `%` as unit, but RN for web supports [`%`, `pt`, `px`, `em`, `rem`, `vh`, `vw`] So what do you think, add all of this as ALLOWED_UNITS or limit support only for `%` for all platforms?

Reply:
In such a case I would go for conditional contents of allowed units.
Something like this maybe:
```
const ALLOWED_UNITS = Platform.OS === 'web' ? ['%', 'pt' ...] : ['%'];
```